src/plugins/pillbox-menus/components/Menu.less

```diff
@@ -110,6 +110,73 @@
     }
   }
 
+  .dsm-settings-color-list-container {
+    display: flex;
+    flex-direction: column;
+
+    .flex {
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  .dsm-settings-color-list {
+    display: flex;
+    flex-direction: column;
+    border-radius: 10px;
+    padding: 0;
+    overflow: hidden;
+    border: 1px solid black;
+    min-width: 100px;
+    flex-grow: 3;
+
+    .dsm-settings-color-list-item-container {
+      position: relative;
+      display: flex;
+
+      .add-remove-buttons {
+        display: none;
+      }
+
+      &:hover .add-remove-buttons {
+        position: absolute;
+        display: block;
+        right: 0;
+        top: 0;
+        background-color: #ffffffaa;
+
+        .dsm-btn-icon {
+          width: 30px;
+          margin-top: -3px;
+        }
+      }
+    }
+
+    input[type="color"] {
+      border: none;
+      margin: 0;
+      padding: 0;
+      outline: none;
+      appearance: none;
+      width: 100%;
+    }
+
+    // for some reason these only work if separated
```

Comment:
Weird. I would chalk it up to one of them erroring taking down the whole rule, but they're both `-webkit`, so neither should error.

Reply:
Yeah tbh I don't remember exactly why it was broken but I think the actual reason was unrelated.